RecoTracker/IterativeTracking/python/iterativeTkConfig.py

```diff
@@ -80,6 +93,16 @@
     minNumberOfLayersWithMeasBeforeFiltering = 0,
 )
 
+#Phase2 : configuring the phase2 track Cluster Remover
+from RecoLocalTracker.SubCollectionProducers.phase2trackClusterRemover_cfi import *
+__trackClusterRemoverBase_Phase2PU140 = phase2trackClusterRemover.clone(
+    maxChi2                                  = cms.double(9.0),
+    phase2pixelClusters                      = cms.InputTag("siPixelClusters"),
+    phase2OTClusters                         = cms.InputTag("siPhase2Clusters"),
+    TrackQuality                             = cms.string('highPurity'),
+    minNumberOfLayersWithMeasBeforeFiltering = cms.int32(0),
```

Comment:
Avoid explicit types as/if the parameters are already defined in the cfi (to enable typo checking).